app/views/shared/_header.html.erb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
```

Comment:
I've removed that line from the partial. We'll see if Hound dislikes it, but we could pass it anyway since it doesn't do much.